src/Mgate/PubliBundle/Controller/DocumentController.php

```diff
@@ -65,24 +66,18 @@ public function voirAction(Document $documentType)
 
     /**
      * @Security("has_role('ROLE_SUIVEUR')")
+     * @param Request $request
+     * @param Etude $etude
+     * @return bool|\Symfony\Component\HttpFoundation\RedirectResponse|\Symfony\Component\HttpFoundation\Response
```

Comment:
Hey :)

Thank you for reviewing that PR and these advices. 

Done in 07f617a8944128bbbfd042d5adc50a9b499c4e66